uint8 color_index
float64[] color
float64[] position
float64 q